sdk/eventhub/event-hubs/samples/expressSample/src/index.ts

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT Licence.
```

Comment:
```suggestion
/*
  Copyright (c) Microsoft Corporation.
  Licensed under the MIT Licence.

  This sample demonstrates how to send events to Event Hubs
  from an express service. The service will take the HTTP body of
  any request sent to `POST /ingest` and transform it before sending
  it to Event Hubs.
  
 As events are handed to the `AsyncBatchingProducer` via the `send()` call,
 the producer will ensure that events are sent in the same batch so long as:
 1. The batch has enough space for additional events.
 2. the maxBatchSize is not exceeded by adding an event.
 3. The elapsed time since the last batch was sent does not exceed the maxWaitTimeInSeconds.
 Once any of these conditions are met, a new batch is created and the cycle continues.
*/
```